### DISCUSSION TITLE
How to hide or remove Organize... bar in Explorer?

### DISCUSSION THREAD

**Discussion by thepurpleone8** [Open-Shell/Open-Shell-Menu#875] [Q&A] [answered]
Hi everyone, I have a question. How can I hide or remove this Organize - System properties -... bar?
![00](https://user-images.githubusercontent.com/92384736/136979102-da931057-dbba-444a-b9eb-022dc4f40599.png)
Thanks in advance!

**Comment by Gaurav-Original-ClassicShellTester** [comment]
You didn't mention your version of Windows and what modifications you have already done to Explorer?

**Reply by thepurpleone8** [reply]
Oh sorry, I'm running Windows 10 version 1803 and I have OpenShell & Classic Explorer installed with Windowblinds theme applied.

**Reply by thepurpleone8** [reply]
Oh and also I have used oldnewexplorer

**Comment by Gaurav-Original-ClassicShellTester** [comment, marked as answer]
This is not related to Open Shell, unsupported if as it can break Explorer badly but this is known to work: https://www.askvg.com/how-to-make-folder-band-auto-hidden-in-windows-vista/ if you use OldNewExplorer: https://msfn.org/board/topic/170375-oldnewexplorer-119/ to disable the Ribbon.

Please confirm and report back if it worked for you.

**Comment by thepurpleone8** [comment]
Alright it worked really well for me, it wasn't related to Open Shell but it was the OldNewExplorer, thank you really much for helping <3
![image](https://user-images.githubusercontent.com/92384736/137122095-423b256b-f8a5-48c3-848a-63ef9717ae2d.png)
